tests/explorer.api.tests/QueryTests.cs

```diff
@@ -374,11 +375,15 @@ public Result FromJsonArray(ref Utf8JsonReader reader)
                     reader.Read();
                 }
 
-                return new Result();
+                return new Result(42);
             }
 
             public struct Result
             {
+                public Result(int dummy)
+                {
+                    _ = dummy;
+                }
```

Comment:
Just curious why you added this - are you concerned the method would be erased by the compiler or something?

Reply:
I was lazy when I tried to fix an analyzer warning and I didn't read the documentation to understand the problem (https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1129.md).
I properly fixed it now.